### PR TITLE
oneof float support

### LINKED ISSUE #82: Bool values or OneOf
Like so many other faker libraries, is it possible to generate fake bool values and OneOf.

I think that bool is easy to understand, but with OneOf, I was searching for something like this

```
faker:"oneof="foo","bar","x","y"
```

thanks

### PR DESCRIPTION
**Pull request by andrew-werdna**
- add support for float32
- ensure support for both float32 and float64
- lint fixes
- add tests for float64 & add test for nil pointer""

fix #82

### CODE REVIEW

**Comment by andrew-werdna** [comment]
@bxcodec I got bored haha. I decided to jump in sooner rather than later with support for the float types.

**Comment by andrew-werdna** [comment]
@bxcodec okay I think it is now ready for review again! 👍

File: tag_argument_extractor.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+	toRet := floatValues[rand.Intn(len(floatValues))]
+	return toRet, nil
+}
+
+func extractInt64FromTagArgs(args []string) (interface{}, error) {
+	bytes := 64
+	var floatValues []int64
+	for _, i := range args {
+		k := strings.TrimSpace(i)
+		j, err := strconv.ParseInt(k, 0, bytes)
```
Comment (bxcodec) [inline tag_argument_extractor.go:45]: any particular reasons to put the base to zero (0) instead of directly `10`? Is it because you want to make this support hex decimal as well?

Comment (bxcodec) [reply tag_argument_extractor.go:45]: I'm sorry for the late reply, so busy lately 🙏

Comment (andrew-werdna) [reply tag_argument_extractor.go:45]: sure

Comment (andrew-werdna) [reply tag_argument_extractor.go:45]: actually I'm running out of steam and am getting more busy over here as well. I'm going to switch the base back to base 10 and leave adding hex/binary/octal/etc other base number types for another PR.
